common/source/docs/common-planning-a-mission-with-waypoints-and-events.rst

```diff
@@ -159,8 +159,25 @@ There is also a Copter-specific :ref:`Copter Mission Command List <copter:missio
 Mission Reset
 =============
 
+On Copter and Rover, the mission is reset to the beginning of the mission list every disarm. On Plane, it is reset similarly, except in AUTO mode which does not reset on disarm and leaves the "mission pointer" on the last mission item to be performed. For all vehicles, any NAV_LAND mission item leaves the "mission pointer" pointing to itself when landing is completed. The :ref:`MIS_OPTIONS<MIS_OPTIONS>` parameter, bit 2, can be set to continue onward in the mission list after landing.
+
 One can set an ``RCx_OPTION`` switch function ("24") to reset the mission item pointer to the beginning of the mission list at any time.
 
+Also, the mission can be set to reset upon any entry into AUTO mode with the :ref:`MIS_RESTART<MIS_RESTART>` parameter. Default (0) is to resume the mission when re-entering AUTO mode.
+
+:ref:`MIS_OPTIONS<MIS_OPTIONS>`
+===============================
+
+The :ref:`MIS_OPTIONS<MIS_OPTIONS>` parameter bits can be set to modify how missions function:
+
+==================================   ================================
+ref:`MIS_OPTIONS<MIS_OPTIONS>` bit   Option
+==================================   ================================
+0                                    Clears the loaded mission on boot. Must be loaded after each boot.
```

Comment:
I guess it is OK to list the bit number because most GCSs will do this for them.  Another option is to provide the actual value in brackets.  Just an idea, not a blocker.